pkg/executables/kubectl.go

```diff
@@ -1311,3 +1311,27 @@ func (k *Kubectl) GetDaemonSet(ctx context.Context, name, namespace, kubeconfig
 
 	return obj, nil
 }
+
+func (k *Kubectl) GetPackagesFromKubectl(ctx context.Context, packageInstanceName, kubeconfig string, output string) (string, error) {
+	params := []string{
+		"get", "packages", kubeconfigFlag, kubeconfig, namespaceFlag, constants.EksaPackagesName,
```

Comment:
is packages a CRD?

Reply:
Yes there is a CRD associated with it but this will be a CR that will be fetched.